test/config.py

```diff
@@ -34,6 +34,7 @@
 IS_WINDOWS = "windows" in _sys or ("win" in _sys and "darwin" not in _sys)
 IS_LINUX = "linux" in _sys
 IS_OSX = "darwin" in _sys
+IS_UNIX = IS_LINUX or IS_OSX
```

Comment:
I'm sure some people would disagree with this definition of `UNIX` :-P

Reply:
Yeah sure. But it works here and I could not find anything better in the first Google attempt ;-)